### PR TITLE
Newsletter copy changes

### PR DESCRIPTION
**Pull request by davidpmccormick**
References #2431 #2601

Updating copy in the newsletter signup form, and removing the default (normalize) `fieldset` styles.

screen shot 2018-05-16 at 16 10 16

### CODE REVIEW

File: common/views/components/NewsletterSignup/NewsletterSignup.js [hunk 1]
```diff
@@ -20,33 +20,29 @@ type State = {|
...
-    description: `What’s On: Access - highlights events, tours and opportunities to get involved, including BSL, Audio Description and Speech-to-Text events`
+    label: `<span class="${font({s: 'HNL5'})}"><span class="${font({s: 'HNM5'})}">Access</span> events, tours and opportunities to get involved, including British Sign Language, Audio Description and Speech-To-Text activities. Sent quarterly with occasional updates.</span>`,
+    name: 'addressbook_40129'
   },
   {
     id: 'young_people_14-19',
-    label: `What’s On for 14-19 year olds`,
-    name: 'addressbook_40132',
-    description: `What’s On for 14-19 year olds: creative opportunities and events for young people aged 14-19, including RawMinds and Saturday Studio`
+    label: `<span class="${font({s: 'HNL5'})}">Events and opportunities to get involved for <span class="${font({s: 'HNM5'})}">14-to-19-year-olds</span>, including RawMinds and Saturday Studios. Sent monthly with occasional updates.</span>`,
```
Comment (jamesgorrie) [inline common/views/components/NewsletterSignup/NewsletterSignup.js]: Why can't we set the font on `<b>` to `HNM5`?

File: common/views/components/NewsletterSignup/NewsletterSignup.js [hunk 2]
```diff
@@ -100,32 +96,35 @@ class NewsletterSignup extends Component<Props, State> {
...
             <p><a href='/whats-on'>Browse our current and upcoming exhibitions and events</a>.</p>
-          </Fragment>
+          </div>
         }
 
         {isError &&
-          <Fragment>
-            <h1>There was a problem</h1>
+          <div className='body-text'>
+            <h1>Sorry, there&rsquo;s been a problem</h1>
```
Comment (jamesgorrie) [inline common/views/components/NewsletterSignup/NewsletterSignup.js]: Could we just use standard characters?

File: common/views/components/NewsletterSignup/NewsletterSignup.js (hunk 1 above)
Comment (davidpmccormick) [reply common/views/components/NewsletterSignup/NewsletterSignup.js]: The browser bolding is inconsistent (and looks crummy at the best of times):

screen shot 2018-05-17 at 11 43 51

File: common/views/components/NewsletterSignup/NewsletterSignup.js (hunk 2 above)
Comment (davidpmccormick) [reply common/views/components/NewsletterSignup/NewsletterSignup.js]: Yep (I thought React was going to complain about html entities needing to be escaped, but this only appears to be true for `'`, and not `’`. Will update

**Pull request merged**